scripts/build_upload.py

```diff
@@ -381,7 +381,7 @@ def upload_anaconda(token, dev):
 
 @upload_wrapper('pypi')
 def upload_pypi():
-    run("twine register upload dist/*")
+    run("twine upload dist/*")
```

Comment:
I just realized I think this can be made slightly better and more bulletproof if we use the exact actual version:
```
run("twine upload dist/bokeh-%s.tar.gz" % CONFIG.version)```